composable/assetlist.json

```diff
@@ -229,6 +229,326 @@
       "logo_URIs": {
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/tinkernet/images/tnkr.svg"
       }
+    },
+    {
+      "description": "The native token of Ethereum, bridged via IBC.",
+      "denom_units": [
+        {
+          "denom": "eth-wei",
+          "exponent": 0
+        },
+        {
+          "denom": "eth",
+          "exponent": 18
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/4C14B42547F6428A4E84F501D21CE5DF6C40C537635A8B0E6A5ED8FB4919BCE7",
+      "name": "Ethereum",
+      "display": "eth",
+      "symbol": "ETH",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "ethereum",
+            "base_denom": "eth-wei"
```

Comment:
None of the base_denoms specified are registered as `base` in ethereum assetlist

Reply:
There are a few that already exist, I will submit the rest of them now

Reply:
is it possible to register two `base` fields under one asset?

Reply:
no, there is only 1, being it's contract